transform/task_test.go

```diff
@@ -1,53 +1,68 @@
 package transform
 
 import (
-	"io/ioutil"
+	"fmt"
+	"io"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 )
 
-func TestTask(t *testing.T) {
-	d := t.TempDir()
-	for _, s := range []sourceType{venue, motive} {
-		ls, err := PathsForSource(s, filepath.Join("..", "testdata"))
-		if err != nil {
-			t.Errorf("expected no error finding paths for %s, got %s", string(s), err)
-		}
-		for _, f := range ls {
-			b, err := ioutil.ReadFile(f)
+func TestTaskRun(t *testing.T) {
+	testdata := []struct {
+		desc   string
+		source sourceType
+		nFiles int
+	}{
+		{"ESTABELE", venue, 1},
+	}
+	for _, test := range testdata {
+		t.Run(test.desc, func(t *testing.T) {
```

Comment:
Como `string(venue) == "ESTABELE"`, podemos elimitar o `test.desc`? Ou você acha que deixa a lógica do teste mais complexa, menos explícita?

```suggestion
		source sourceType
		nFiles int
	}{
		{venue, 1},
	}
	for _, test := range testdata {
		t.Run(string(test.source), func(t *testing.T) {
```

Reply:
Pode mudar sim. Fica a vontade.

Outra coisa, se não teremos outros casos, talvez seja melhor tirar todo o maquinário em torno de tabe driven tests e voltar para o jeito que você tinha sugerido no começo.